examples/marathon.l5d

```diff
@@ -0,0 +1,17 @@
+namers:
+- kind:      io.l5d.experimental.marathon
+  prefix:    /io.l5d.marathon
+  host:      marathon.mesos
+  port:      80
+  uriPrefix: /marathon
```

Comment:
Good point, I'll update config.md with some of this detail.

That API prefix can vary depending on how marathon is configured. 

For example, running locally based on these instructions:
https://mesosphere.github.io/marathon/docs/
... there is no API prefix, you'd do:
```
$ curl localhost:8080/v2/apps
```

Using the default DCOS / AWS setup:
https://mesosphere.com/amazon/
... you'd do:
```
curl $(dcos config show core.dcos_url)/marathon/v2/apps
```